roseau/load_flow_single/models/branches.py

```diff
@@ -83,7 +83,7 @@ def _check_same_voltage_level(self) -> None:
         if (
             self.bus1._nominal_voltage is not None
             and self.bus2._nominal_voltage is not None
-            and not np.isclose(self.bus1._nominal_voltage, self.bus2._nominal_voltage)
+            and not math.isclose(self.bus1._nominal_voltage, self.bus2._nominal_voltage)
```

Comment:
This is minor but math/cmath `isclose` are 60x faster than numpy's and they work with numpy scalars.